ddtrace/contrib/grpc/constants.py

```diff
@@ -12,7 +12,6 @@
 GRPC_REQUEST_METADATA_PREFIX_KEY = "grpc.request.metadata."
 GRPC_RESPONSE_METADATA_PREFIX_KEY = "grpc.response.metadata."
 GRPC_HOST_KEY = "grpc.host"
-GRPC_PORT_KEY = "grpc.port"
```

Comment:
Confirmed on Metabase that this tag name is not in use by any clients. Good to switch to new tag name.

Reply:
Sounds like I need a more complete view of the complete lifecycle of tags. This one was the last remaining `.port` tag generated by a client process. Thanks for confirming!